src/foam/nanos/auth/User.js

```diff
@@ -904,8 +905,47 @@ foam.CLASS({
       javaCode: `
         HashMap<String, NotificationSetting> settingsMap = new HashMap<String, NotificationSetting>();
 
+        settingsMap = getImpliedNotificationSettings(x);
+        for ( NotificationSetting setting : settingsMap.values() ) {
+          System.err.println(setting.getClassInfo().getId());
+          setting.doNotify(x, this, notification);
+        }
+      `
+    },
+    {
+      name: 'getImpliedNotificationSettings',
+      args: 'Context x',
+      type: 'java.util.HashMap',
+      code: async function(x) {
+        x = x || this.__subContext__;
+        let map = {};
+        // System defaults
+        (await x.notificationSettingDefaultsDAO.where(this.EQ(foam.nanos.notification.NotificationSetting.SPID, '*')).select())?.array?.map(a => {
```

Comment:
Should the DefaultsDAO's be accessible on the client?

Reply:
It is permissioned and permissioned per DAO entry as well, so users have to be explicitly granted access for every operation. Only permission granted is read for specific defaults